game-server/data/handlers/ai/instance/empyreanCrucible/KingConsierdAI.java

```diff
@@ -57,104 +56,69 @@ public void handleAttack(Creature creature) {
 		hpPhases.tryEnterNextPhase(this);
 		if (isHome.compareAndSet(true, false)) {
 			startBloodThirstTask();
-
-			ThreadPoolManager.getInstance().schedule(new Runnable() {
-
-				@Override
-				public void run() {
-					SkillEngine.getInstance().getSkill(getOwner(), 19691, 1, getTarget()).useNoAnimationSkill();
-					ThreadPoolManager.getInstance().schedule(new Runnable() {
-
-						@Override
-						public void run() {
-							SkillEngine.getInstance().getSkill(getOwner(), 17954, 29, getTarget()).useNoAnimationSkill();
-						}
-
-					}, 4000);
-
-				}
-			}, 2000);
+			scheduleInitialSkills();
 		}
 	}
 
+	private void scheduleInitialSkills() {
+		ThreadPoolManager.getInstance().schedule(() -> {
+			getOwner().queueSkill(19691, 1, 0);
```

Comment:
Queueing skills with nextSkillTime=0 does not behave identically to `useNoAnimationSkill`. Have you confirmed this looks and feels right in combat?
This should be checked with all skill IDs used in this handler: 17951, 17952, 17954, 19624, 19690, 19691.